js/components/modal.tsx

```diff
@@ -0,0 +1,36 @@
+import { Modal as RestartModal } from "@restart/ui";
+import { ReactElement } from "react";
+
+export const Modal = ({
+  children,
+  title,
+  show,
+  onHide,
```

Comment:
Might be more clear as `onClose`?

Reply:
I went with `onHide` because that's what Restart UI uses, but I think you're right that `onClose` is clearer. 54f1b2b